package.json

```diff
@@ -147,6 +149,11 @@
         },
         "dir"
       ]
+    },
+    "pkg": {
+      "artifactName": "tidepool-uploader-${version}.${ext}",
+      "allowAnywhere": false,
+      "allowRootDirectory": false
```

Comment:
Oh electron-builder, this is a pretty non-intuitive way to specify home directory only. Would've been nice if JSON supported comments so that we could make this clearer. I guess it's a temporary solution until they fix the permissions issue in Electron itself. I'm just wondering if any clinics have Macs with multiple users set up - seems unlikely.